tests/countries/test_turkmenistan.py

```
@@ -0,0 +1,86 @@
+            self.assertNotEqual(holidays_2014[date(2014, 10, 6)], "Day of Remembrance")
+    
+        holidays_2015 = Turkmenistan(years=2015)
+        self.assertIn(date(2015, 10, 6), holidays_2015)
+        self.assertEqual(holidays_2015[date(2015, 10, 6)], "Day of Remembrance")
+
+    
+    def test_neutrality_day_changes(self):
+        """Test Neutrality Day date changes."""
+        self.assertIn(date(2017, 12, 12), Turkmenistan(years=2017))
+        self.assertEqual("Neutrality Day", Turkmenistan(years=2017)[date(2017, 12, 12)])
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)